UnitTests/expected/Strings-File-Dot-Syntax-Swift3.swift.out

```diff
@@ -0,0 +1,79 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+import Foundation
+
+// swiftlint:disable file_length
+// swiftlint:disable type_body_length
+// swiftlint:disable nesting
+// swiftlint:disable variable_name
+// swiftlint:disable valid_docs
+
+struct L10n {
+  /// Title of the alert
+  static let AlertTitle = L10n.tr("alert_title")
+  /// Some alert body there
+  static let AlertMessage = L10n.tr("alert_message")
+  /// Hello, my name is %@ and I'm %d
+  static func Greetings(p0: String, p1: Int) -> String {
+    return L10n.tr("greetings", p0, p1)
+  }
+  /// These are %3$@'s %1$d %2$@.
+  static func ObjectOwnership(p0: Int, p1: String, p2: String) -> String {
+    return L10n.tr("ObjectOwnership", p0, p1, p2)
+  }
+
+  struct Apples {
+    /// You have %d apples
+    static func Count(p0: Int) -> String {
+      return L10n.tr("apples.count", p0)
+    }
+  }
+
+  struct Bananas {
+    /// Those %d bananas belong to %@.
+    static func Owner(p0: Int, p1: String) -> String {
```

Comment:
This is where the problem lies. We should adapt the template so that it should instead generate this:
```
static func owner(_ p0: Int, _ p1: String) -> String
```

Not sure how I missed that…